dhis-2/dhis-support/dhis-support-hibernate/src/main/java/org/hisp/dhis/hibernate/jsonb/type/JsonBinaryType.java

```diff
@@ -101,9 +107,36 @@
     }
 
     @Override
-    public boolean equals( Object x, Object y ) throws HibernateException
+    public boolean equals( Object x, Object y )
+            throws HibernateException
```

Comment:
Why can equals throw a HibernateException?

Reply:
because this is not Object.equals(), rather it extends org.hibernate.usertype.equals(x,y) which is defined throwing Exception, so it was already there. 
I agree however that exception is never thrown, so I'll remove it from the method signature.